keyboards/han60/readme.md

```diff
@@ -0,0 +1,28 @@
+# han60

```

Comment:
Please use Unix line endings for all files.

Reply:
Do you mean `\n` rather than `\`?

Reply:
LF rather than CRLF.